lib/drivers/medtronicDriver.js

```diff
@@ -325,9 +327,11 @@ module.exports = function (config) {
 
         var medtronicMessage = packet.slice(MEDTRONIC_PACKET_START);
         console.log('Current CBG page:', common.bytes2hex(medtronicMessage));
-        var currentPage = struct.extractBEInt(medtronicMessage,0);
-        var glucose = struct.extractByte(medtronicMessage,5);
-        var isig = struct.extractByte(medtronicMessage,7);
+        // pump stores a maximum of 32 glucose and ISIG history pages
+        var MAX_CBG_PAGES = 32;
+        var currentPage = Math.min(struct.extractBEInt(medtronicMessage,0), MAX_CBG_PAGES) + 1;
+        var glucose = Math.min(struct.extractByte(medtronicMessage,5), MAX_CBG_PAGES) + 1;
+        var isig = Math.min(struct.extractByte(medtronicMessage,7), MAX_CBG_PAGES) + 1;
```

Comment:
They're the page counters, so I've renamed `glucose` to `glucosePage` and `isig` to `isigPage`.